packages/account/src/Components/formik-confirmation-checkbox/formik-confirmation-checkbox.tsx

```diff
@@ -0,0 +1,60 @@
+import React from 'react';
+import { Checkbox, Text } from '@deriv/components';
+import { isEmptyObject, isMobile } from '@deriv/shared';
+import { useFormikContext } from 'formik';
+
+/**
+ * Props for the confirmation checkbox component.
+ */
+type TConfirmationCheckboxProps = {
+    /**
+     * A boolean value indicating whether the checkbox is confirmed or not.
+     */
+    confirmed: boolean;
+    /**
+     * A React.SetStateAction function to set the confirmed state of the checkbox.
+     */
+    setConfirmed: React.Dispatch<React.SetStateAction<boolean>>;
+    /**
+     * The label of the checkbox.
+     */
+    label: string;
+};
+
+/**
+ * A checkbox component for confirming an action with an optional description.
+ *
+ * This component renders a checkbox that can be used to confirm an action, such as agreeing to terms
+ * and conditions. It also allows displaying an optional description next to the checkbox.
+ *
+ * @name FormikConfirmationCheckbox
+ * @returns {JSX.Element} React component that renders a checkbox with a label
+ */
+export const FormikConfirmationCheckbox = ({
```

Comment:
can we just use same approach as we do for other checkboxes? for example in ClosingAccountReasonFrom.
just wrapping Checkbox with Field, rather than implement additional useState with additional flag?

Reply:
We can definitely modify the component as we need. 
ClosingAccountReasonForm also has `onChangeCheckbox` prop

Reply:
so if we really need this component, maybe we should move it to 'form' folder in 'components'? and should we mention 'formik' in the file name? how do you think?